product_docs/docs/pem/9/monitoring_event_history.mdx

```diff
@@ -0,0 +1,14 @@
+| user_name     | This field provides the username who executed or scheduled the event.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| component     | This field provides the event name.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| operation     | This field provides the action taken for the event. For example, if the user has enabled an alert blackout, it displays `enable_alert_blackout`.                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| message       | This field provides the description of the action taken for the event. For example, if the user has enabled an alert blackout for the server, it displays `Enabled the alert blackout for the server`.                                                                                                                                                                                                                                                                                                                                                                                |
+| details       | This fields provides the arguments along with its value: <br/><br/>- `AlertBlackoutValue` - displays value `true` if an alert blackout is enabled or disabled.<br/><br/>- `IsAgent` - displays value `true` if alert blackout is for the agent and `false` if alert blackout is for the server.<br/><br/>- `Ids` - diplays the ID of the agent or the server. <br/><br/>- `Scheduled` - displays the value `true` if it is scheduled and `false` if it is configured manually. <br/><br/>- `JobID` - displays the `JobID` if it is scheduled and `null` if it is configured manually. |
```

Comment:
You can write like:-
This field provides the JSON output for the particular event that happened.
I guess need not to describe the JSON keys inside it to make this doc more generic.
@asheshv What are your thoughts?